modules/cognitive-services/account/README.md

```diff
@@ -875,3 +702,179 @@ module account './cognitive-services/account/main.bicep' = {
+        "keyVaultProperties": {
+            "identityClientId": "12345678-1234-1234-1234-123456789012", // ID must be updated for new identity
+            "keyVaultUri": "https://adp-[[namePrefix]]-az-kv-nopr-002.vault.azure.net/",
+            "keyName": "keyEncryptionKey",
+            "keyversion": "1111111111111111111111111111111" // Version must be updated for new keys
+        }
+    }
+}
+// With service-managed key
+"encryption": {
+    "value": {
+        "keySource": "Microsoft.CognitiveServices"
+    }
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+// With customer managed key
+encryption: {
+    keySource: 'Microsoft.KeyVault'
+    keyVaultProperties: {
+        identityClientId: '12345678-1234-1234-1234-123456789012' // ID must be updated for new identity
+        keyVaultUri: 'https://adp-[[namePrefix]]-az-kv-nopr-002.vault.azure.net/'
+        keyName: 'keyEncryptionKey'
+        keyversion: '1111111111111111111111111111111' // Version must be updated for new keys
+    }
+}
+// With service-managed key
+encryption: {
+    keySource: 'Microsoft.CognitiveServices'
+}
+```
+
+</details>
+<p>
+### Parameter Usage: `roleAssignments`
+
+Create a role assignment for the given resource. If you want to assign a service principal / managed identity that is created in the same deployment, make sure to also specify the `'principalType'` parameter and set it to `'ServicePrincipal'`. This will ensure the role assignment waits for the principal's propagation in Azure.
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"roleAssignments": {
+    "value": [
+        {
+            "roleDefinitionIdOrName": "Reader",
+            "description": "Reader Role Assignment",
+            "principalIds": [
+                "12345678-1234-1234-1234-123456789012", // object 1
+                "78945612-1234-1234-1234-123456789012" // object 2
+            ]
+        },
+        {
+            "roleDefinitionIdOrName": "/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11",
+            "principalIds": [
+                "12345678-1234-1234-1234-123456789012" // object 1
+            ],
+            "principalType": "ServicePrincipal"
+        }
+    ]
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+roleAssignments: [
+    {
+        roleDefinitionIdOrName: 'Reader'
+        description: 'Reader Role Assignment'
+        principalIds: [
+            '12345678-1234-1234-1234-123456789012' // object 1
+            '78945612-1234-1234-1234-123456789012' // object 2
+        ]
+    }
+    {
+        roleDefinitionIdOrName: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11'
+        principalIds: [
+            '12345678-1234-1234-1234-123456789012' // object 1
+        ]
+        principalType: 'ServicePrincipal'
+    }
+]
+```
+
+</details>
+<p>
+
+### Parameter Usage: `networkAcls`
```

Comment:
Is part of the common test & can be removed

Reply:
removed. Regenerated readme